docs/reference/search/request/highlighting.asciidoc

```diff
@@ -8,15 +8,19 @@ The following is an example of the search request body:
 
 [source,js]
 --------------------------------------------------
+GET /_search
 {
-    "query" : {...},
+    "query" : {
+        "match_all": {}
```

Comment:
I'd use a `match` query here because it gives the highlighter something to think about. I know it doesn't matter because we don't assert anything about the result but it is still nice.